mmv1/products/cgc/terraform.yaml

```diff
@@ -55,6 +55,15 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         primary_resource_id: "custom_hostname_instance"
         vars:
           custom_hostname_instance_name: "custom-hostname-instance-name"
+    ### Compute Reservation Local ###
+      - !ruby/object:Provider::Terraform::Examples
+        name: "compute_reservation"
+        primary_resource_type: "google_compute_reservation"
+        primary_resource_id: "gce_reservation_local"
+        vars:
+          gce_reservation_local: "gce-reservation-local"
+        test_env_vars:
+          project: :PROJECT_NAME
```

Comment:
I think the quickest, easiest thing to do would be to add `skip_import_test: true` here.